allensdk/brain_observatory/behavior/criteria.py

```diff
@@ -0,0 +1,216 @@
+"""
+Functions for calculating mtrain state transitions.
+If criteria are met, return true. Otherwise, return false.
+"""
+
```

Comment:
I think generally, I might consider renaming these functions so that they have a consistent convention, especially as they all return bools.

Maybe something similar to?
```
has_good_dprime_over_last_three_days()
has_good_dprime_yesterday()
has_no_response_bias()
```

Reply:
I agree that the names aren't great. However, they correspond exactly to the name of transitions in the mtrain state machine, so in that sense they are self-documenting. I'm refactoring the code so they're no longer necessary to be the same, but I thought keeping the names that way was clearer for the use-case.